homeassistant/const.py

```diff
@@ -423,6 +423,7 @@
 PRESSURE_PA: str = "Pa"
 PRESSURE_HPA: str = "hPa"
 PRESSURE_BAR: str = "bar"
+PRESSURE_CBAR: str = "cbar"
```

Comment:
I don't think we should add another pressure unit to the common constants. On the contrary I think we should consolidate the different pressure units we report in.

That's obviously out of scope of this PR.

Reply:
Moved to onewire\const.py